components/script/devtools.rs

```diff
@@ -66,8 +66,9 @@ pub fn handle_evaluate_js(global: &GlobalScope, eval: String, reply: IpcSender<E
             assert!(rval.is_object());
 
             rooted!(in(*cx) let obj = rval.to_object());
-            let class_name = CStr::from_ptr(ObjectClassName(*cx, obj.handle()));
-            let class_name = str::from_utf8(class_name.to_bytes()).unwrap();
+            /*let class_name = CStr::from_ptr(ObjectClassName(*cx, obj.handle()));
+            let class_name = str::from_utf8(class_name.to_bytes()).unwrap();*/
+            let class_name = "fixme";
```

Comment:
Please file an issue and reference it from here at least?

Reply:
Hmm, why is this needed?